app/controllers/messages_controller.rb

```diff
@@ -1,9 +1,9 @@
 class MessagesController < ApplicationController
   def create
-    message = Message.new(message_params.merge(source: current_node.name))
-    
+    message = Message.new(message_params.merge(source: Redis.current.get('node_name')))
```

Comment:
current_node never worked for me - that's why i looked for the node manually - but yes - it can be extracted to a single place in code